src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import './ButtonPanel.css';
+import Button from '../Button/Button';
+
+const ButtonPanel = () => (
+  <div className="panel-container">
+    <div className="row">
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="AC"
+      />
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="+/-"
+      />
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="%"
+      />
+      <Button
+        wide="1"
+        name="/"
+      />
+    </div>
+    <div className="row">
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="7"
+      />
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="8"
+      />
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="9"
+      />
+      <Button
+        wide="1"
+        name="*"
+      />
+    </div>
+    <div className="row">
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="4"
+      />
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="5"
+      />
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="6"
+      />
+      <Button
+        wide="1"
+        name="-"
+      />
+    </div>
+    <div className="row">
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="1"
+      />
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="2"
+      />
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="3"
+      />
+      <Button
+        wide="1"
+        name="+"
+      />
+    </div>
+    <div className="row">
+      <Button
+        color="#e0e0e0"
+        wide="2"
+        name="0"
+      />
+      <Button
+        color="#e0e0e0"
+        wide="1"
+        name="."
+      />
+      <Button
+        wide="1"
+        name="="
+      />
+    </div>
+  </div>
+);
+
```

Comment:
You should probably also assign a *DEFAULT PROP* value for `color` inside the COLOR component ([inside color.js](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)), so that you dont repeat yourself here

Reply:
Just to a reminder. You can't pass zero because in proptypes it supposed to be **BOOLEAN**

Reply:
am not talking about ZERO, am talking about the `wide` property.....which is a boolean and it's supposed to be passed to the ZERO button, which is the button that supposed to be wide since it takes more space

Reply:
I misunderstood. I got it.